client/src/components/TaskList/TaskList.js

```diff
@@ -3,23 +3,41 @@ import React from "react";
 import Paper from "@material-ui/core/Paper";
 import Grid from "@material-ui/core/Grid";
 import Button from "@material-ui/core/Button";
+import Fade from "@material-ui/core/Fade";
 import Typography from "@material-ui/core/Typography";
 import { Draggable, DragDropContext, Droppable } from "react-beautiful-dnd";
 
 import { TaskObj } from "../../classes/TaskObj.js";
 import Task from "../Task/Task.js";
 import { ObjArrayCopy } from "../../common/ObjArrayCopy.js";
 
-const contents_start = [new TaskObj("Content 1", 0), new TaskObj("Content 2", 0), new TaskObj("Content 3", 0), new TaskObj("Content 4", 0)];
+const contents_start = [new TaskObj("Content 1", 0), new TaskObj("Content 1", 0), new TaskObj("Content 1", 0), new TaskObj("Content 2", 0), new TaskObj("Content 3", 0), new TaskObj("Content 4", 0)];
 
 export default function TaskList() {
     //Set contents from DB as React state
     const [contents, setContents] = React.useState(contents_start);
     const [dragging, setDragging] = React.useState(false);
+    const toDelete = React.useRef(false);
 
     //Tried useState before, but timer would double start on the first start, requireing a hacky fix. Found to use useRef instead
     const timerIDStates = React.useRef(contents.map((item) => item.timerID));
 
+    function addNewTask() {
+        setContents(prevContents => {
+            const newContents = ObjArrayCopy(prevContents);
+            newContents.push(new TaskObj("", 0));
+
+            //Delay until after setContent is complete
+            setTimeout(() => {
+                //Scroll to end of task list
+                const elem = document.getElementById("task-list-paper");
+                elem.scrollTop = elem.scrollHeight;
+            }, 0);
+
+            return newContents;
+        });
+    }
+
     React.useEffect(() => {
```

Comment:
@bangyii  why do we need this when we already have line25-44 in PlayPauseButton.js?